functional-tests/src/test/java/helm/BaseHelmChartTest.java

```diff
@@ -10,6 +10,7 @@
 import com.oracle.bedrock.OptionsByType;
 import com.oracle.bedrock.deferred.options.InitialDelay;
 import com.oracle.bedrock.deferred.options.RetryFrequency;
+import com.oracle.bedrock.deferred.PermanentlyUnavailableException;
```

Comment:
I do not see this import being used. Also should we check for PermanentlyUnavailableException instead of RuntimeException

Reply:
as we discussed, catching RuntimeException since that is exception that is documented to be thrown by Application.waitFor method. Removed the old unused import.